tests/Integration/Behaviour/Features/Context/Domain/Product/AbstractProductFeatureContext.php

```diff
@@ -116,6 +117,48 @@ protected function assertStringProperty(ProductForEditing $productForEditing, ar
         }
     }
 
+    /**
+     * @param ProductForEditing $productForEditing
+     * @param array $data
+     * @param string $propertyName
+     */
+    protected function assertNumberProperty(ProductForEditing $productForEditing, array &$data, string $propertyName): void
```

Comment:
We might need to move all these `assert...Property()` helpers into a dedicated Util/Helper class one day

Reply:
yep but in another PR as these methods, for now, are tightly related to `ProductForEditing `

Reply:
Please add a todo :)